src/main/java/seedu/duke/logic/commands/RouteMapCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.duke.logic.commands;
+
+import seedu.duke.BusRoute;
+import seedu.duke.exceptions.CustomException;
+import seedu.duke.exceptions.ExceptionType;
+import seedu.duke.ui.Ui;
+
+import java.util.Scanner;
+
+public class RouteMapCommand extends Command{
+    /*External classes used*/
+    private static final Scanner userInput = new Scanner(System.in);
+
+    @Override
+    public void executeCommand(){
+        try {
+            Ui.printRouteSelectionMessage();
+            String userBusRouteSelection = userInput.nextLine().trim().toLowerCase();
```

Comment:
I was wondering if the getCommand() of Ui can be used here. Just to avoid declaring a separate Scanner.

Reply:
Thank you! I will work on that in my next commit